packages/kit/src/exports/vite/index.js

```diff
@@ -330,10 +330,11 @@ function kit() {
 			completed_build = false;
 
 			if (is_build) {
-				rimraf(paths.build_dir);
+        if (!vite_config.build.watch) {
+				  rimraf(paths.build_dir);
+				  rimraf(paths.output_dir);
+        }
```

Comment:
lint is failing. you need to use tabs instead of spaces
```suggestion
					if (!vite_config.build.watch) {
						rimraf(paths.build_dir);
						rimraf(paths.output_dir);
					}
```